NuRadioMC/utilities/medium.py

```diff
@@ -65,6 +330,26 @@ def __init__(self):
             z_0 = 77.*units.meter, 
             delta_n = 0.423,
             )
+        
+class southpole_2015_up(medium_base.IceModelSimple):
```

Comment:
Do you need these somewhere?

Reply:
No, this was only for cross check reasons, I deleted it.